apps/root/src/pages/home/components/portfolio/index.tsx

```diff
@@ -255,7 +257,7 @@ const Portfolio = ({ selectedWalletOption }: PortfolioProps) => {
     const mappedBalances = map(Object.entries(tokenBalances), ([key, value]) => ({
       ...value,
       key,
-      relativeBalance: ((value.balanceUsd || 0) / assetsTotalValue.wallet) * 100,
+      relativeBalance: ((value.balanceUsd || 0) / assetsTotalValue.wallet) * 100 || 0,
```

Comment:
I would avoid 0/0 division, so it would be okay with a ternary to check assetsTotalValue.wallet.
`assetsTotalValue.wallet && value.balanceUsd ? ((value.balanceUsd / assetsTotalValue.wallet) * 100) : 0`

Reply:
@th0rOdinson sounds about right. done.